eng/Version.Details.xml

```diff
@@ -122,17 +122,17 @@
     </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="9.0.0-beta.24416.2">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="9.0.0-beta.25407.2">
```

Comment:
Updated Arcade because versions of the packages that it depends on (NuGetRepack, SourceLink) are no longer available from the feed. Matches the version used in the 17.12 branch